charts/keycloakx/templates/statefulset.yaml

```diff
@@ -148,6 +148,7 @@ spec:
             {{- with .Values.extraPorts }}
             {{- toYaml . | nindent 12 }}
             {{- end }}
+          {{- if .Values.health.enabled }}
```

Comment:
I'd say we can skip this if clause - if someone decides to stick to the "old" livness-probes, he might not need to enable the health endpoint. But, with the proposal, he needs to enable the endpoint in order to use live-, readiness and startupProbe at all. What do you think?

Reply:
I don't see the use case here. Why should anyone want to use the old health checks?

Reply:
The health endpoint is enabled by default. This PR is about using health checks at all. It doesn't matter in this case if the old health checks are configured or not.

Reply:
In that matter correct - but there would be a dependency between `health.enabled` and the usage of livenessProbe etc. 
I agree, that the use case for not using health for healthchecks is very abstract and i cannot think of any valid reason.. so let's merge.